libs/model/src/chain-events/ChainEventCreated.command.ts

```diff
@@ -1,19 +1,23 @@
 import {
-  EvmCommunityStakingEventSignatures,
```

Comment:
we already talked about this, just a reminder that this is not a command - we'll rename when we define a better framework for integration policies